.github/workflows/openqa.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Run a basic openQA test
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+  # Note how this runs on:pull_request_target and not on:pull_request!
+  # The difference is that this runs always with the context of the master
+  # branch. This is necessary to allow accessing the API credential secrets.
+  workflow_dispatch:
+env:
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+  OPENQA_API_KEY: ${{ secrets.OPENQA_API_KEY }}
+  OPENQA_API_SECRET: ${{ secrets.OPENQA_API_SECRET }}
+  GH_REPO: ${{ github.event.pull_request.head.repo.full_name }}
+  GH_REF: ${{ github.event.pull_request.head.ref }}
+
+jobs:
+  trigger_and_monitor_openqa:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/tumbleweed:client
+    steps:
+      - uses: actions/checkout@v3
+      - name: Determine the latest Tumbleweed build on o3
+        id: latest_build
+        run: >-
+          echo build=$(curl -s
+          ${OPENQA_HOST:-https://openqa.opensuse.org}/group_overview/${OPENQA_GROUP_ID:-1}.json
+          | jq -r '([ .build_results[] | select(.tag.description=="published") | select(.version=="Tumbleweed") | .build ] | sort | reverse)[]'
+          | head -n1) >> "$GITHUB_OUTPUT"
+      - name: Trigger and monitor a basic openQA test on o3
+        run: >-
+          openqa-cli schedule
+          --monitor
+          --host "${OPENQA_HOST:-https://openqa.opensuse.org}/"
+          --apikey "$OPENQA_API_KEY" --apisecret "$OPENQA_API_SECRET"
+          --param-file SCENARIO_DEFINITIONS_YAML=scenario-definitions.yaml
+          DISTRI=openSUSE VERSION=Tumbleweed FLAVOR=dev ARCH=x86_64
```

Comment:
```suggestion
          DISTRI=openSUSE VERSION=Tumbleweed FLAVOR=GITHUB ARCH=x86_64
```

Reply:
Why the ALL CAPS? :)

Reply:
> Why the ALL CAPS? :)

Totally arbitrary, maybe if we scream at the openQA jobs, they will work faster?